libraries/lib-exceptions/AudacityException.h

```diff
@@ -214,7 +214,7 @@ R GuardedCall(
          // At this point, e is the "current" exception, but not "uncaught"
          // unless it was rethrown by handler.  handler might also throw some
          // other exception object.
-         if (!std::uncaught_exception()) {
+         if (std::uncaught_exceptions() == 0) {
```

Comment:
A build-breaker, as you know, until we make a commitment to raising the minimum macOS version.  So please omit this commit from the branch.